src/tests/key-protect.cpp

```diff
@@ -75,13 +75,11 @@ TEST_F(rnp_tests, test_key_protect_load_pgp)
 
         pgp_key_t *tmp = NULL;
         assert_non_null(tmp = rnp_tests_get_key_by_id(ks, keyids[0], NULL));
-        assert_non_null(tmp);
 
         // steal this key from the store
-        key = (pgp_key_t *) calloc(1, sizeof(*key));
+        key = new pgp_key_t();
```

Comment:
Style nit: I almost never see parens when there are no constructor args, but we don't have a rule on this

Reply:
@dewyatt Thanks, fixing this.